frontend/js/components/environments/Results.js

```diff
@@ -31,7 +31,9 @@ class Results extends Component {
     }
 
     componentWillMount() {
```

Comment:
`componentWillMount` is called on the server and in the client.  `componentDidMount` is only called in the client.  Did you mean to put this in that hook?